doc/cookbook/monolog_request_hash.rst

```diff
@@ -0,0 +1,84 @@
+Adding a request hash using Monolog
+==============================
```

Comment:
Sorry about that. Please let me know if there's something else that needs tweaking before it can be merged ;)